main/src/main/resources/org/clulab/numeric/HOLIDAY.tsv

```diff
@@ -0,0 +1,49 @@
+#
+# List of holidays in the United States extracted from the Jollyday api.
+# This list can be obtained with the following snippet:
+#
+#   import de.jollyday.{HolidayManager, HolidayCalendar, ManagerParameters}
+#   import collection.JavaConverters._
+#   val holidayManager = HolidayManager.getInstance(ManagerParameters.create(HolidayCalendar.UNITED_STATES))
+#   holidayManager.getCalendarHierarchy.getChildren.keySet.asScala.
+#                   map(holidayManager.getHolidays(2021, _).asScala).flatten.map(_.getDescription)
+#
+#
+Lincoln's Birthday
+Easter Monday
+Washington's Birthday
+Labour day
+César Chávez's Birthday
+Seward's Day
+Christmas Eve
+Mardi Gras
+Battle of Bennington
+Arbor Day
+West Virginia Day
+Election day
+Confederate Memorial Day
+Statehood day
+Truman Day
+Christmas
+Presidents Day
+Prince Jonah Kuhio Kalanianaole Day
+Independence Day
+New Years Eve
```

Comment:
I'm sure that apostrophes will enjoy free variation throughout.  I would have to write New Year's Eve.  Will we match ok on 's, s', or just s?

Reply:
It might not be something we can solve, but perhaps we should have an idea of what happens.  I don't know offhand how the tokenization works in these cases.  We might get a separate 's as a token.

Reply:
I've checked and I need to separate the 's in the lexicon, e.g. "Lincoln's Birthday" -> "Lincoln 's Birthday". 

"New Year's Eve" is not recognized. I can add apostrophe alternatives to the lexicon and follow the same "comment as canonical name" I propose to include "MLK Day".